packages/orb-sync-lib/src/database/postgres.ts

```diff
@@ -25,6 +25,7 @@ export class PostgresClient {
     const chunkSize = 5;
     const results: pg.QueryResult<T>[] = [];
 
+    console.time('upsert-many');
```

Comment:
multiple upsert-many invocations could run in parallel, leading to weird timing side effect - you can generate a uuid or something else here to make it unique and use it in both calls to ensure it uses the same identifier